api/policy.go

```diff
@@ -57,7 +57,7 @@ type NewPolicy struct {
 	Limit         int      `json:"limit,omitempty" yaml:"limit,omitempty"`
 	EvalFrequency string   `json:"evalFrequency,omitempty" yaml:"evalFrequency,omitempty"`
 	AlertEnabled  bool     `json:"alertEnabled" yaml:"alertEnabled"`
-	AlertProfile  string   `json:"alertProfile" yaml:"alertProfile"`
+	AlertProfile  string   `json:"alertProfile,omitempty" yaml:"alertProfile"`
```

Comment:
nit.  should we add this for yaml as well for consistency?

Reply:
Conflating `"alertProfile": ""` with `"alertProfile": null` and the absence of `"alertProfile"` in the json object seems odd to me as I would expect the customer to get an error if they add `"alertProfile": ""` to the input. That said, this will work and solve the current issue, so 🚢

Reply:
@hazedav I didn't know if we wanted YAML but, since you suggest it I will add it 😄